src/components/PostDetails.tsx

```diff
@@ -5,59 +5,36 @@ import { NewCommentForm } from './NewCommentForm';
 import * as commentsApi from '../api/comments';
 
 import { Post } from '../types/Post';
-import { Comment, CommentData } from '../types/Comment';
+import { CommentData } from '../types/Comment';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
+import {
+  fetchComments,
+  addComment,
+  removeComment,
+  removeAllComments,
+} from './commentsSlice/commentsSlice';
 
 type Props = {
   post: Post;
 };
 
 export const PostDetails: React.FC<Props> = ({ post }) => {
-  const [comments, setComments] = useState<Comment[]>([]);
-  const [loaded, setLoaded] = useState(false);
-  const [hasError, setError] = useState(false);
+  const [error, setError] = useState(false);
```

Comment:
You have a state for error and has flag in store ( hasError). What is the difference and is it possible to get by with one value?

Reply:
mystery)